File: app/src/main/java/com/example/tiktacticssignup_login/LauncherActivity.java
package com.example.tiktacticssignup_login;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import androidx.activity.EdgeToEdge;
import androidx.appcompat.app.AppCompatActivity;

public class LauncherActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        EdgeToEdge.enable(this);
        setContentView(R.layout.activity_launcher);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent(LauncherActivity.this, Login.class);
                startActivity(intent);

                overridePendingTransition(R.anim.fade_in, R.anim.fade_out);

                finish();
            }
        }, 3000);
    }
}

File: app/src/main/java/com/example/tiktacticssignup_login/LauncherActivity.kt
package com.example.tiktacticssignup_login;



import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity

class LauncherActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        enableEdgeToEdge()
        setContentView(R.layout.activity_launcher)

        Handler().postDelayed({
                val intent = Intent(this, Login::class.java)
        startActivity(intent)

        overridePendingTransition(R.anim.fade_in, R.anim.fade_out)

        finish()
        }, 3000)
    }
}

